### DISCUSSION TITLE
The Centering Jedi

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#16] [Galaxy Trivia] [answered]
Your `<div>` is training to be a Jedi and wants to learn how to center itself both vertically and horizontally using the Force (or CSS). What’s the secret technique?

**Comment by bicced** [comment, marked as answer]
display: flex;
align-items: center;
justify-content: center;

**Reply by meviru** [reply]
That is correct! Keep it up! 👏👏
